tests/README.md

```diff
@@ -34,7 +34,9 @@
 
 1. Run `make dm_integration_test_build` to generate DM related binary for integration test
 
-2. Run `make integration_test` to execute the integration tests. This command will
+2. Setup two MySQL servers with binlog enabled first and set `GTID_MODE=ON`, export proper environment variable `MYSQL_HOST1`, `MYSQL_PORT1`, `MYSQL_PASSWORD1` and `MYSQL_HOST2`, `MYSQL_PORT2`, `MYSQL_PASSWORD2`, default is `127.0.0.1`, `3306`, `123456` and `127.0.0.1`, `3307`, `123456`.
```

Comment:
How about using a _table_ format to show these variables and their default value (ditto for L58)?